### PR TITLE
Handle os path get request

### PR DESCRIPTION
**Pull request by yibane-yibane**

### CODE REVIEW

File: data_base/SQLAlchemy.py [hunk 1]
```diff
@@ -49,3 +41,24 @@ def is_data_model_exists(self, data_model: DB.db.Model, where_section: dict):
...
+
+    def get_entities(self, data_model, select_section: list, attributes_filters: dict):
+        query = self._create_query(data_model, select_section, attributes_filters)
+
+        return query.all()
+
+    def get_entity(self, data_model, select_section: list, attributes_filters: dict):
+        query = self._create_query(data_model, select_section, attributes_filters)
+
+        return query.first()
```
Comment (MayoG) [inline data_base/SQLAlchemy.py]: Change to one method instead of duplicating code

File: data_base/DataModelTransactions.py [hunk 2]
```diff
@@ -49,10 +49,19 @@ def create_and_insert_metadata_and_set_metadata_id_to_frames(frames: list, image
...
             metadata = data_models.Metadata(fov=fov, azimuth=azi, elevation=elev, tag=tag)
             db.insert_one(metadata)
 
         frames[index].set_metadata_id(metadata.id)
+
+
+def get_videos_path():
+    db = SQLAlchemyHandler()
+    return db.get_entities(data_model=data_models.Video,
+                           select_section=['id', 'os_path'],
```
Comment (MayoG) [inline data_base/DataModelTransactions.py]: Why returning the ids of the paths ?

File: flask_app/FlaskApp.py [hunk 3]
```diff
@@ -18,9 +19,14 @@ def __init__(self):
         self.azure_container_handler = AzureBlobContainerHandler()
 
     def run(self):
+        self.app.add_url_rule('/get_videos_path', view_func=self.get_videos_path, methods=['GET'])
         self.app.add_url_rule('/upload_video', view_func=self.upload_video, methods=['POST'])
         self.app.run()
 
+    def get_videos_path(self):
+        videos_path_dict = dict(DataModelTransactions.get_videos_path())
+        return json.dumps({'videos_path': videos_path_dict}), 200, {'ContentType': 'application/json'}
```
Comment (MayoG) [inline flask_app/FlaskApp.py]: change to plural `'videos_paths'`

**Review by MayoG — CHANGES_REQUESTED** [review]
Add docs to you new methods

File: data_base/SQLAlchemy.py (hunk 1 above)
Comment (yibane-yibane) [reply data_base/SQLAlchemy.py]: `get_entities`  returns `query.all()` and `get_entity` returns `query.first()`

Why it's a duplicatin?

Comment (MayoG) [reply data_base/SQLAlchemy.py]: The functions both get the same parameters and execute the same line:
`query = self._create_query(data_model, select_section, attributes_filters) `

File: data_base/SQLAlchemy.py [hunk 4]
```diff
@@ -49,3 +41,24 @@ def is_data_model_exists(self, data_model: DB.db.Model, where_section: dict):
             exists_query = exists_query.where(data_model.__dict__[field] == value)
 
         return self.db.session.query(exists_query).scalar()
+
+    def get_entities(self, data_model, select_section: list, attributes_filters: dict, count):
```
Comment (MayoG) [inline data_base/SQLAlchemy.py]: Add docs please,
You can add default values to you methods, default value for count here will be great

File: data_base/DataModelTransactions.py [hunk 5]
```diff
@@ -49,10 +49,21 @@ def create_and_insert_metadata_and_set_metadata_id_to_frames(frames: list, image
...
 
         frames[index].set_metadata_id(metadata.id)
+
+
+def get_videos_paths():
+    db = SQLAlchemyHandler()
+    return db.get_entities(data_model=data_models.Video,
+                           select_section=['os_path'],
+                           attributes_filters={},
+                           count=None)
```
Comment (MayoG) [inline data_base/DataModelTransactions.py]: If u decided to say that None means all please specify that in the docs.

**Review by MayoG — CHANGES_REQUESTED** [review]

**Review by MayoG — APPROVED** [review]

**Pull request merged**